plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -54,29 +55,39 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 );
 
-const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
-  if (about) {
-    let formatted = `OS: ${about.operatingSystem}\nResources: ${about.resourceUtilization}\nnode: ${about.nodeJsVersion}\nbackstage: ${about.backstageVersion}\nDependencies:\n`;
-    const deps = about.dependencies;
-    for (const key in deps) {
-      if (Object.prototype.hasOwnProperty.call(deps, key)) {
-        formatted = `${formatted}    ${deps[key].name}: ${deps[key].versions}\n`;
-      }
-    }
-    window.navigator.clipboard.writeText(formatted);
-  }
-};
-
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const alertApi = useApi(alertApiRef);
+  const [infoCopied, setInfoCopied] = useState(false);
   const { about, loading, error } = useInfo();
+  const [, setCopied] = useCopyToClipboard();
```

Comment:
Would it make sense to use the `state` like they do in the docs - https://github.com/streamich/react-use/blob/master/docs/useCopyToClipboard.md#usage - and then use the `alertApi` to log the error?